src/routes/sessions/[id]/+page.svelte

```diff
@@ -52,14 +54,17 @@
 
 	const shouldConfirmDeletion = writable(false);
 
-	$: session = loadSession(data.id);
 	$: isNewSession = !session?.messages.length;
 	$: isLastMessageFromUser = session?.messages[session.messages.length - 1]?.role === 'user';
 	$: session && scrollToBottom();
 	$: if ($settingsStore?.ollamaModel) session.model = $settingsStore.ollamaModel;
 	$: knowledge = knowledgeId ? loadKnowledge(knowledgeId) : null;
 	$: shouldFocusTextarea = !isPromptFullscreen;
 
+	afterNavigate(() => {
+		session = loadSession(data.id);
+	});
+
```

Comment:
Was there a particular reason for this change?
If not let's revert it and also revert the change on line 43.

Reply:
There was a reason for the way I was handling the title before, but now it is not necessary anymore.

This way (using `afterNavigate`) is how knowledge is handled, though. However, I have already restored it to the previous version.